docx/oxml/bookmark.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Custom element classes for bookmarks
+"""
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from . import parse_xml
+from .ns import nsdecls, qn
+
+from .xmlchemy import (BaseOxmlElement, OneOrMore,
+    RequiredAttribute, ZeroOrOne)
+
+from .simpletypes import ST_RelationshipId, ST_String
+from docx.oxml import OxmlElement
+
+class CT_bookmarkStart(BaseOxmlElement):
+    """
+    ``<w:bookmarkStart>`` element
+    """
+    bookmark = OneOrMore('w:bookmarkStart', ST_String)
+    name = RequiredAttribute('w:name', ST_String)
+    bmrk_id = RequiredAttribute('w:id', ST_RelationshipId)
+
+        
+    def set_id(self, id):
+        self.set(qn('w:id'), str(id))
+        
+    def set_name(self, name):
+        self.set(qn('w:name'), name)
+    
+    def _bookmark(self):
+        return OxmlElement('w:bookmarkStart')       
+
+class CT_bookmarkEnd(BaseOxmlElement):
+    """
+    The ``<w:bookmarkEnd>`` element
+    """
+    bookmark = OneOrMore('w:bookmarkEnd', ST_String)
+    bmrk_id = RequiredAttribute('w:id', ST_RelationshipId)
+    
+    def set_id(self, id):
+        self.set(qn('w:id'), str(id))
+
+    def _bookmark(self):
+        return OxmlElement('w:bookmarkEnd')          
+      
+    
```

Comment:
* There should never be white-space at the end of a file (e.g. blank line(s)). My editor (Vim) has automatic checking for this sort of thing, yours might too. What editor do you use?
* I've recently adopted PEP 257 https://www.python.org/dev/peps/pep-0257/, which governs the formatting of docstrings. The short story is that class docstrings should appear like this:

  ```python
  class CT_BookmarkStart(BaseOxmlElement):
      """`w:bookmarkStart` element

      This is a further elaboration of the docstring, often not needed for custom element classes.
      """

      def ...(...):
          ...
  ```

  The first line must be short enough to fit in a line length of 77, (as all code here) The rest of the docstring, if present should appear separated by a single blank line. Then there should be a blank line between the docstring and the first definition (often `def __init__()` in non-element classes).

  This is another case where I haven't made the change globally, but respect the standard for all new work and when I'm editing a class (or sometimes just in the neighborhood, but never when there aren't other changes to the file).